app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -174,36 +151,12 @@ class TopicRevisionFragmentTest {
     }
   }
 
-  @Test
-  fun testTopicRevisionFragment_loadFragment_checkSpanCoun_isTwo() {
```

Comment:
Why is this test being removed?

Reply:
This is shifted to its related *spanTest file.

Reply:
`TopicRevisionSpanTest`